js/main.js

```diff
@@ -27,12 +28,12 @@ function contentLoaded(event) {
 
 function switchView(view) {
   var $dataView = event.target.getAttribute('data-view');
```

Comment:
Hmmm, you shouldn't be using the `event` object in this function. You want to just pass in a string of the name of the view to show then loop over the list of views to find the one that matches that name